.gitignore

```diff
@@ -10,3 +10,4 @@ coverage
 gemfiles/*.lock
 Gemfile.lock
 spec/examples.txt
+.idea
```

Comment:
I'm not sure about this, since there are so many editors each with their own extra files. I *think* you can create a global git ignore file to help ignore your editor stuff everywhere. Would that work for you?

Reply:
It's easy to add this and they do in other cucumber projects, so just copied it from there.

Reply:
Ok, let's keep it then.